examples/trace/build.gradle

```diff
@@ -29,5 +29,5 @@ dependencies {
     compile(libraries.opentelemetry_api)
     compile(libraries.opentelemetry_sdk)
     compile(libraries.google_cloud_trace)
-    compile(libraries.opentelemetry_operations_java)
```

Comment:
@zoercai I missed that you already fixed that in https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/pull/59, will rebase if that one gets merged first.

Reply:
Done.